flink-runtime/src/main/scala/org/apache/flink/runtime/messages/JobManagerMessages.scala

```diff
@@ -19,11 +19,14 @@
 package org.apache.flink.runtime.messages
 
 import org.apache.flink.runtime.accumulators.AccumulatorEvent
+import org.apache.flink.runtime.client.JobStatusMessage
 import org.apache.flink.runtime.executiongraph.{ExecutionAttemptID, ExecutionGraph}
 import org.apache.flink.runtime.instance.{InstanceID, Instance}
 import org.apache.flink.runtime.jobgraph.{JobGraph, JobID, JobStatus, JobVertexID}
 import org.apache.flink.runtime.taskmanager.TaskExecutionState
 
+import scala.collection.JavaConverters._
```

Comment:
There are also explicit imports of ```JavaConverters._``` in ```RegisteredTaskManagers``` which then should be removed if putting the import here.